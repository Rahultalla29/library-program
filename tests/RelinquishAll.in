RELINQUISH ALL 111111
ADD MEMBER Sasuke Uchiha
RELINQUISH ALL 111111
ADD COLLECTION sample1.csv
RENT 100000 111111
RENT 100000 111112
RENT 100000 111113
RELINQUISH ALL 100000
MEMBER HISTORY 100000
EXIT
