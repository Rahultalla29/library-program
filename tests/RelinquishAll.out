user: No members in system.

user: Success.

user: No such member in system.

user: 17 books successfully added.

user: Success.

user: Success.

user: Success.

user: Success.

user: To Kill a Mockingbird (Harper Lee)
The Hobbit (J.R.R. Tolkien)
The Iliad (Homer)

user: Ending Library process.
